src/depackers/stb_vorbis.c

```diff
@@ -741,12 +1121,14 @@ static int compute_codewords(Codebook *c, uint8 *len, int n, uint32 *values)
       }
 
       while (z > 0 && !available[z]) --z;
-      if (z == 0) { /*assert(0);*/ return FALSE; }
+      if (z == 0) { return FALSE; }
       res = available[z];
+      assert(z >= 0 && z < 32);
       available[z] = 0;
       add_entry(c, bit_reverse(res), i, m++, len[i], values);
-      // propogate availability up the tree
+      // propagate availability up the tree
       if (z != len[i]) {
+         assert(len[i] < 32);
```

Comment:
why not return failure somehow and assert() instead?

Reply:
This seems like a duplicate of the one below, marking resolved...